tests/unit/test_config.py

```diff
@@ -25,31 +25,83 @@
 
 import pytest
 from flexmock import flexmock
-from jsonschema.exceptions import ValidationError
-
 from ogr.abstract import GitProject, GitService
+
 from packit.actions import ActionName
 from packit.config import (
     JobConfig,
     PackageConfig,
-    TriggerType,
+    JobTriggerType,
     SyncFilesConfig,
     SyncFilesItem,
     get_packit_config_from_repo,
     Config,
+    JobType,
+    JobNotifyType,
 )
+from packit.exceptions import PackitInvalidConfigException
+
+
+def get_job_config_dict_simple():
+    return {"job": "build", "trigger": "release", "notify": []}
+
+
+def get_job_config_dict_full():
+    return {
+        "job": "propose_downstream",
+        "trigger": "pull_request",
+        "notify": ["pull_request_status"],
+        "metadata": {"a": "b"},
+    }
+
+
+@pytest.fixture()
+def job_config_dict_simple():
+    return get_job_config_dict_simple()
+
+
+@pytest.fixture()
+def job_config_dict_full():
```

Comment:
I don't see where are these two fixtures used.

Reply:
yeah, I was trying to unify the configs and came to a state where they are probably indeed not needed: it's a pity that you can't use parametrize with fixtures